packages/manager/tools/webpack-config/src/webpack.common.js

```diff
@@ -23,9 +24,21 @@ const cacheLoader = {
   },
 };
 
+const readClosestPackage = (entry) => {
```

Comment:
Duplicate changes introduced here:
https://github.com/ovh/manager/pull/8855#pullrequestreview-1340812036

Reply:
It will be fix on main branch.